src/components/MailboxThread.vue

```diff
@@ -429,7 +430,18 @@ export default {
 		margin: 0 !important;
 	}
 }
-
+:deep(.app-content-list) {
```

Comment:
there are now two deep selectors for the same element. would it make sense to consolidate?